tests/h/search/query_test.py

```diff
@@ -513,11 +513,41 @@ def search(self, search):
 
 
 class TestUsersAggregation(object):
+    def test_it_aggregates_annotations_on_users(self, Annotation, search):
+        for i in range(2):
+            Annotation(userid="acct:pa@example.com")
+        Annotation(userid="acct:pb@example.com")
 
-    @pytest.fixture
-    def search(self, search):
         search.append_aggregation(query.UsersAggregation())
-        return search
+        result = search.run({})
+
+        users_results = result.aggregations.get('users')
+        count_pa = next(r for r in users_results if r["user"] == "acct:pa@example.com")['count']
+        count_pb = next(r for r in users_results if r["user"] == "acct:pb@example.com")['count']
+
+        assert len(users_results) is 2
```

Comment:
`==` rather than `is` should be used for comparing values for _equality_ rather than _identity_.  This code _happens_ to work because Python interns (re-uses the same object) for small integer values, but that is not guaranteed AFAIK.

The main cases to use `is` are: `assert x is None|True|False` or `assert x is y` where "x" and "y" are objects and you really want them to be the _same_ object and not just equivalent.